protected/humhub/libs/SelfTest.php

```diff
@@ -316,22 +316,6 @@ public static function getResults()
             ];
         }
 
-        // Checks GraphicsMagick Extension
-        $title = 'PHP - ' . Yii::t('AdminModule.information', '{phpExtension} Extension', ['phpExtension' => 'GraphicsMagick']);
```

Comment:
Is support for GraphicsMagick being dropped?

Reply:
We will only support it unofficially for now.